pkg/helper/helpers.go

```diff
@@ -88,3 +91,17 @@ func UnstructuredDuration(in *metav1.Duration) string {
 	}
 	return in.ToUnstructured().(string)
 }
+
+func FindFilter(name string) bool {
+	var cfg config.FrontendConfig
+	err := yaml.Unmarshal(config.StaticFrontendConfig, &cfg)
```

Comment:
this unmarshal is called repeatedly (in `for` loops in `validateFlowMetric`), can we try have a single call for it?